diff -ur dxpc-3.7.0.orig/main.C dxpc-3.7.0/main.C
--- dxpc-3.7.0.orig/main.C	Wed Aug  6 16:02:39 1997
+++ dxpc-3.7.0/main.C	Thu Jul 13 20:18:03 2000
@@ -537,7 +537,7 @@
 #if defined(DXPC_ACCEPT_IS_SIZE_T)
 	  size_t addrLen = sizeof(sockaddr_in);
 #else
-	  int addrLen = sizeof(sockaddr_in);
+	  socklen_t addrLen = sizeof(sockaddr_in);
 #endif /* defined(DXPC_ACCEPT_IS_SIZE_T) */
 	  int newFD = accept(tcpFD, (sockaddr *) & newAddr, &addrLen);
 	  if (newFD == -1)
@@ -554,7 +554,7 @@
 #if defined(DXPC_ACCEPT_IS_SIZE_T)
 	  size_t addrLen = sizeof(sockaddr_un);
 #else
-	  int addrLen = sizeof(sockaddr_un);
+	  socklen_t addrLen = sizeof(sockaddr_un);
 #endif /* defined(DXPC_ACCEPT_IS_SIZE_T) */
 	  int newFD = accept(unixFD, (sockaddr *) & newAddr, &addrLen);
 	  if (newFD == -1)
@@ -668,7 +668,7 @@
 #if defined(DXPC_ACCEPT_IS_SIZE_T)
       size_t addrLen = sizeof(sockaddr_in);
 #else
-      int addrLen = sizeof(sockaddr_in);
+      socklen_t addrLen = sizeof(sockaddr_in);
 #endif /* defined(DXPC_ACCEPT_IS_SIZE_T) */
       int newFD = accept(proxyFD, (sockaddr *) & newAddr, &addrLen);
       if (newFD == -1)
